pages/about.html

```diff
@@ -4,9 +4,76 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About me</title>
+   <link href="../stylesheets/style.css" rel="stylesheet" />
 </head>
-<body>
-  
+<body class="background-about">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" />
+  <header>
+    <nav class="nav">
+      <ul class="nav-item">
+        <li class="scaler"><a href="index.html" target = "_self">Home</a></li>
+        <li class="scaler"><a href="about.html" target = "_self">About me</a></li>
+        <li class="scaler"><a href="portfolio.html" target = "_self">Projects</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main class="container-about">
+    <section class="personal-info">
+      <img class = "avatar" src="../images/profile.jpeg" alt="Avatar">
+      <aside class ="name-pronouns">
+        <h1>Karina Gorkava</h1>
+        <p id="pronouns"> she/they</p>
+      </aside>
+      <aside class ="contact-info">
+        <p class="decorator-profile">::: get in touch</p>
+        <p> <a href = "mailto: infernormal@gmail.com">Email: infernormal@gmail.com</a></p>
```

Comment:
No space in a mailto link.

```suggestion
        <p> <a href = "mailto:infernormal@gmail.com">Email: infernormal@gmail.com</a></p>
```